app/Models/HandleReservation.php

```diff
@@ -44,8 +44,17 @@ class HandleReservation extends ApiModel
         'reason' => 'sometimes|string|max:255',
     ];
 
-    public static function findAll(): Collection
+    public static function findAll(bool $activeOnly = false): Collection
     {
-        return HandleReservation::all()->sortBy('handle', SORT_NATURAL)->values();
+        $rows = HandleReservation::query();
+        if ($activeOnly) {
+            $today = date("Y-m-d");
```

Comment:
The query can be simplified to just check for to see if a handle entry has not expired. While the the start date is definitely useful on when a handle was entered, the VCs do not deal with reserved handles which might take effect in the future.

Reply:
One other thing, you can use `now()` in place of `date("Y-m-d")`. Both the database framework and engine will correctly deal with the carbon object.

Reply:
looks good! Done